### ISSUE TITLE
Display count of records

### ISSUE THREAD

**Issue by asantaniello** [olifolkerd/tabulator#3106]
Apologies if I've missed this, but is there a config in Tabulator to display the count of records in the table, both in the current page, and for all pages (where pagination is applied)?
Something like "100 records" or "Showing 40-50 of 100 records" in the footer or header, à la jquery DataTables:   https://www.datatables.net/
If not, adding this would be very helpful. In many places in my projects, it is the only reason I have not yet replaced DataTables with Tabulator yet.

I don't see it in the Tabulator examples, and am surprised this isn't an already existing or requested feature.

**Comment by LilDevil** [comment]
Oli may have a different opinion, but it seems to me this is something that should be outside of Tabulator. Every project would probably want it displayed differently, if displayed at all.

You can use the `dataLoaded`, `dataChanged` and/or the `dataFiltered` callbacks. 

In your callback function use `table.getDataCount()`, `table.getPage()` and `table.getPageMax()` to get the info you want. Then format it and stuff it into a display element.

**Comment by huyz** [comment]
Thanks to the above tips, this is the code that worked for me:
```js
  updateRecordCount() {
    if (!this.#table) {
      // Tabulator may be in the process of initial loading and sending out spurious events
      return;
    }
    const that = this;
    // For some reason Tabulator sends the dataFiltered event before it's actually done updating
    // everything; so we have to get to the back of the event queue.
    setTimeout(function() {
      const activeCount = that.#table.getDataCount("active");
      const totalCount = that.#table.getDataCount();
      $(".RecordCount__active").text(activeCount.toLocaleString());
      $(".RecordCount__total").text(totalCount.toLocaleString());
    }, 0);
  }

```

**Comment by olifolkerd** [comment]
Hey All,

I have just pushed an update to the 5.1 branch that adds the `paginationCounter` option that allows for display of a configurable pagination counter.

This will be included in the 5.1 release.

Cheers

Oli :)
